storm-core/src/clj/backtype/storm/daemon/supervisor.clj

```diff
@@ -671,6 +787,11 @@
                         0
                         (conf SUPERVISOR-HEARTBEAT-FREQUENCY-SECS)
                         heartbeat-fn)
+    (doseq [storm-id downloaded-storm-ids] (add-blob-references (:localizer supervisor) storm-id
+                                             conf))
```

Comment:
doseq body should have its own line. Then conf can be moved back into the same line as the function it is being passed to.